src/views/Home.vue

```diff
@@ -13,7 +13,22 @@
       <div class="rf-col-8">
 
         <IntroSection></IntroSection>
-        <SearchBar></SearchBar>
+
+        <SearchBar>
+             <form>
+              <div class="rf-search-bar" id="search-input">
+                  <label class="rf-label" for="search-input-input">Label de la barre de recherche</label>
```

Comment:
Ce label est invisible pour nous, mais lu par un lecteur d'écran. Mieux vaut utiliser un texte signifiant. Par exemple : 

```suggestion
                  <label class="rf-label" for="search-input-input">Rechercher un financement</label>
```